src/applications/vaos/sass/vaos.scss

```diff
@@ -1,5 +1,5 @@
-@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
-@import "../../../platform/forms/sass/m-schemaform";
+@import '~@department-of-veterans-affairs/formation/sass/shared-variables';
```

Comment:
Looks like these still need to be switched back to `"`.

Reply:
Darn, I missed the import files.